tsconfig.json

```
@@ -21,6 +21,6 @@
 		},
 		"incremental": true
 	},
-	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/lib/bugsnag.tjs"],
```

Is tj supposed to be wrapped in `[]`?
```suggestion
	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/lib/bugsnag.[tj]s"],
```

---

IDK this is added automatically